main.go

```diff
@@ -154,7 +156,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	kongCtx.FatalIfErrorf(kongCtx.Run(ctx, client))
+	err = kongCtx.Run(ctx, client)
+	if err != nil {
+		if k8serrors.IsForbidden(err) {
+			fmt.Println("nctl: Permission denied: are you part of the org?	")
```

Comment:
I think this will print out a message, but the exit code of `nctl` will be 0, which wouldn't be correct I think. Maybe you could use another function of `kongCtx` to exit and print the generic permission error?